tests/models/test_queryparams.py

```diff
@@ -87,6 +87,11 @@ def test_empty_query_params():
     assert str(q) == "a="
 
 
+def test_invalid_query_params():
+    with pytest.raises(TypeError):
```

Comment:
Seems nice to ensure that the formatted message is correct
```suggestion
    with pytest.raises(TypeError, match=r"Expected str, int, float, bool, or None\. Got 'bytes'\."):
```

Reply:
That's neat, thanks.

Reply:
Sorry I broke your line length :)

Reply:
🤨